lib/server/index.js

```diff
@@ -52,6 +52,25 @@ class GameServer {
     process.stdout.write(`Connection from ${socket.conn.remoteAddress}\n`);
   }
 
+  listGames() {
```

Comment:
You can already iterate `server.games` where instance of `GameServer` is available.